db/db.js

```diff
@@ -147,5 +154,5 @@ knex.closeDb = function () {
   })
 }
 
-// knex.ensureSchema().then(knex.closeDb);
+knex.ensureSchema().then(knex.closeDb);
```

Comment:
This need to be commented out; otherwise it will run this function and close db connection immediately after starting the node server @pschreibs85